src/components/Dropdown/DropdownNativeSelect.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import styles from './DropdownNativeSelect.st.css';
+import dropdownStyles from './Dropdown.st.css';
+import buttonStyles from '../Button/Button.st.css';
+import { DATA_HOOKS, ICON_SIZE } from './constants';
+import { ReactComponent as ArrowIcon } from '../../assets/icons/CaretDown.svg';
+import { PRIORITY, SIZE } from '../Button';
+import classNames from 'classnames';
+import { DropdownOptionProps } from './DropdownOption';
+import { TPAComponentsContext } from '../TPAComponentsConfig';
+import { TPAComponentProps } from '../../types';
+import { TooltipSkin } from '../Tooltip/TooltipEnums';
+import { ReactComponent as ErrorIcon } from '../../assets/icons/Error.svg';
+import { Tooltip } from '../Tooltip';
+
+interface DropdownNativeSelectProps {
+  onSelect(selectedOption: DropdownOptionProps): void;
+  selectedOption: DropdownOptionProps;
+  options: DropdownOptionProps[];
+  placeholder: string;
+  disabled: boolean;
+  error?: boolean;
+  errorMessage?: string;
+  'aria-label'?: string;
+  'aria-labelledby'?: string;
+}
+
+export class DropdownNativeSelect extends React.Component<
+  DropdownNativeSelectProps & TPAComponentProps
+> {
+  static contextType = TPAComponentsContext;
+
+  private onSelect(e) {
+    const { selectedOption: prevSelectedOption } = this.props;
+    const optionId = e.target.value;
+    if ((prevSelectedOption && prevSelectedOption.id) === optionId) {
+      return;
+    }
+    const selectedOption = this.props.options.find(({ id }) => id === optionId);
+    this.props.onSelect(selectedOption);
+  }
+
+  private renderOptions(hasPlaceholder: boolean) {
+    const { placeholder, options } = this.props;
+    return (
+      <>
+        {hasPlaceholder && (
+          <option data-hook={DATA_HOOKS.placeholderOption} value="" disabled>
+            {placeholder}
+          </option>
+        )}
+        {options.map((option, i) => (
+          <option key={i} value={option.id} disabled={!option.isSelectable}>
```

Comment:
Shouldn't the selected option have `selected={option.selected}`, or something like that?

Reply:
I've read this: https://reactjs.org/docs/forms.html#the-select-tag
And I'm still not sure if it needs to be added. 
The selection(show which option is selected) is working by default with <select> so I'm not sure what is the added value

Reply:
the difference between using the `selected` attribute vs. not using it,
is having a controlled component vs an uncontrolled component.
In our case, this component is an uncontrolled component, so you're right that maybe in this case (until we merge that PR I made) we don't need it.
However, I think we are using the state's `selectedOption` property for the selected option.
So that the native select element will be aligned to the custom desktop component.

Reply:
Just to be clear - right now we are not using the state's selected option in the native dropdown to find out which of the options is selected. We have the placeholder/ first option as defaults, and after first selection, the <select> is functioning as an uncontrolled component and managing the state by itself

Reply:
yeah, so I was thinking just to have it aligned with the desktop version, maybe it would be best to use it as well.

In addition, the `initialSelectedId` prop should be used for the first option selected.
And some might use it for changing the selected option.

So I guess it's best to keep it aligned so that the behaviour will be identical.

And when i'll migrate it to the controlled version i'll change both

Reply:
What did we decide here?